sdk/keyvault/azure-keyvault-secrets/azure/keyvault/secrets/models.py

```diff
@@ -163,27 +163,39 @@ def tags(self):
         return self._tags
 
 
-class Secret(SecretAttributes):
-    """All a secret's attributes, and its value."""
+class Secret(object):
```

Comment:
We decided all models should have accessors for identifying properties i.e. id, name, vault url and version.

Reply:
added those accessors to Secret and DeletedSercret, thanks!